app/services/rapid_ready_for_decision/fast_track_pdf_generator.rb

```diff
@@ -74,86 +80,21 @@ def add_intro
       end
     end
 
-    def add_blood_pressure_intro
-      header = blood_pressure_data? ? 'One Year of Blood Pressure History' : 'No blood pressure records found'
-      bp_note =
-        blood_pressure_data? ? "<font size='11'>Blood pressure is shown as systolic/diastolic.\n</font>" : ''
-      end_date = @date.strftime('%m/%d/%Y')
-      start_date = (@date - 1.year).strftime('%m/%d/%Y')
-      search_window = "VHA records searched from #{start_date} to #{end_date}"
-      bp_intro_lines = [
-        "<font size='16'>#{header}</font>",
-        "<font size='11'><i>#{search_window}</i></font>",
-        "<font size='11'><i>All VAMC locations using VistA/CAPRI were checked</i></font>",
-        "\n",
-        bp_note
-      ]
-
-      bp_intro_lines.each do |line|
-        @pdf.text line, inline_format: true
-      end
-
-      return @pdf unless blood_pressure_data?
-
-      @pdf.text "\n", size: 10
-    end
-
     def add_blood_pressure_list
-      @blood_pressure_data.each do |bp|
-        systolic = bp[:systolic]['value'].round
-        diastolic = bp[:diastolic]['value'].round
-
-        @pdf.text "<b>Blood pressure: #{systolic}/#{diastolic}</b>",
-                  inline_format: true, size: 11
-        @pdf.text "Taken on: #{bp[:effectiveDateTime].to_date.strftime('%m/%d/%Y')} " \
-                  "at #{Time.iso8601(bp[:effectiveDateTime]).strftime('%H:%M %Z')}",
-                  size: 11
-        @pdf.text "Location: #{bp[:organization] || 'Unknown'}", size: 11
-        @pdf.text "\n", size: 8
-      end
-
-      @pdf.text "\n", size: 12
+      template = File.read('app/services/rapid_ready_for_decision/views/hypertension/blood_pressure_readings.erb')
```

Comment:
Reminder: rename `hypertension` directory

Reply:
Aaah thank you! Good catch!